internal/blsgen/lib.go

```diff
@@ -48,11 +49,44 @@ func WritePrivateKeyWithPassPhrase(privateKey *ffi_bls.SecretKey, passphrase str
 	return fileName
 }
 
+func visit(files *[]string) filepath.WalkFunc {
+	return func(path string, info os.FileInfo, err error) error {
+		if err != nil {
+			log.Fatal(err)
+		}
+		fi, err := os.Stat(path)
+		if !fi.Mode().IsDir() {
+			*files = append(*files, path)
+		}
+		return nil
+	}
+}
+
+// FindBlsPriKey finds bls private key from bls private store with passphrase.
+func FindBlsPriKey(BlsPublicKey, blsPass string, consensusPriKey *ffi_bls.SecretKey) bool {
+	var files []string
+	var err error
+	if _, err = os.Stat(BlsPriKeyStore); os.IsNotExist(err) {
+		return false
+	}
+
+	err = filepath.Walk(BlsPriKeyStore, visit(&files))
```

Comment:
why do we need a walk?  If the file doesn't exist, we should simply exit and print out error?